Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -19,20 +19,20 @@ import BraintreeCore
     // MARK: - Initializers
 
     ///  Creates a SEPA Direct Debit client.
-    /// - Parameter apiClient: An instance of `BTAPIClient`
-    @objc(initWithAPIClient:)
-    public init(apiClient: BTAPIClient) {
-        self.apiClient = apiClient
-        self.sepaDirectDebitAPI = SEPADirectDebitAPI(apiClient: apiClient)
+    /// - Parameter authorization: A valid client token or tokenization key used to authorize API calls.
+    @objc(initWithAuthorization:)
+    public init(authorization: String) {
+        self.apiClient = BTAPIClient(newAuthorization: authorization)
+        self.sepaDirectDebitAPI = SEPADirectDebitAPI(authorization: authorization)
```

Comment:
i wonder if the `SEPADirectDebitAPI(apiClient: apiClient)` can remain the same as before - i think we may only need to update the `BTSEPADirectDebitClient` init but i may be mistaken. cc: @jaxdesmarais

Reply:
Correct, `SEPADirectDebitAPI` is internal so we can keep the signature for that the same.

Reply:
Lets revert the changes in the internal `SEPADirectDebitAPI` as @agedd mentioned and just pass in `apiClient` there. It will be initialized above so can just pass it through.